### PR TITLE
Update usage instructions

### PR DESCRIPTION
**Pull request by camacho**
Update the instructions about how to use the new Framer Actions. Blocked by https://github.com/framer/actions/pull/2

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+This Action is a wrapper for the [Framer CLI](https://www.npmjs.com/package/framer-cli) to build and publish packages to the [Framer Store](https://store.framer.com/).
 
-Whenever a commit is pushed, the GitHub action will build the Framer project. On commits pushed to the `master` branch, the Framer project will additionally be published to the store.
+## Usage
 
-For more documentation about how to configure the Framer CLI, check out the [`framer-cli` package on npm](https://www.npmjs.com/package/framer-cli)
+```workflow
+action "Build" {
+   uses = "framer/actions/bridge@master"
+   args = ["build", <your-project-path.framerfx>]
```
Comment (koenrh) [inline readme.md]:
```suggestion
   args = ["build", "<your-project-path.framerfx>"]
```

File: readme.md [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
+ }
+
+ action "Publish" {
+   uses = "framer/actions/bridge@master"
+   args = ["publish", <your-project-path.framerfx>, "--yes"]
```
Comment (koenrh) [inline readme.md]:
```suggestion
   args = ["publish", "<your-project-path.framerfx>", "--yes"]
```

File: readme.md [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+   needs = ["Build"]
+   secrets = ["FRAMER_TOKEN"]
+}
+```
+
+### Secrets
+
+The `FRAMER_TOKEN` secret is necessary for publishing to the Framer store and can be generated by running:
```
Comment (koenrh) [inline readme.md]: Should we describe how you can add a secret to a GitHub repository?

Or at least add a reference to https://developer.github.com/actions/managing-workflows/storing-secrets/ ?

File: readme.md (hunk 1 above)
Comment (koenrh) [reply readme.md]: To make it clear that the value should be wrapped in `""`?

File: readme.md (hunk 3 above)
Comment (camacho) [reply readme.md]: Good call - linked

**Pull request merged**